### ISSUE TITLE
Named and tamed hamster in wheel despawns

### ISSUE THREAD

**Issue by SumaniPardia** [capnkirok/animaniamod#297]
### Expected Behavior
I place a tamed hamster is a wheel with plenty of food (piped in with a vanilla hopper) and it generates RF until it runs out of food.

### Actual Behavior
Twice I’ve had the hamster go missing/despawn.  One was named and there was no death message and there is no where for anything to spawn that might have damaged it.  It might be related to going to the nether as I noticed both missing shortly after returning from the nether, I will try again tonight in a creative world to confirm.

### Steps to Reproduce
Tame hamster.
Name hamster (optional).
Place a wheel with a hopper pointing in to it.
Add food to hopper.
Place hamster in wheel.
Verify it is generating RF with a battery of some type (I used EnderIO basic capacitor).
Go to nether for several minutes.
When you return there will be no hamster and no RF generated.

### Version of Minecraft, version of Animania, Single Player or Server
Egnimatica 2 - 1.59
Using Forge-1.12.2-14.23.5.2781
Animania-1.12.2-1.6.2

### Screenshots encouraged
Will add tonight

**Comment by katubug** [comment]
We are having a similar issue, although the tamed hamsters are unnamed, and nothing is attached to the wheels.

When they are placed in a wheel, they run for a short while and then disappear. The chunks needs to be unloaded and then reloaded before they reappear - and a player reports that some have not reappeared, but I have yet to test that myself.

**Comment by katubug** [comment]
Just wanted to update this - while we haven't tested it on my server recently, I did some testing just now in singleplayer and could not replicate the issue.

I tamed two hamsters and named them both (Cucumber and Kumquat). I placed a Wheel in the overworld, and added a vanilla Hopper full of Hamster Food attached to it. Placed Kumquat inside, then teleported to the Nether, as the OP said they did. Hung out there for about a minute, then hopped back to the overworld. Kumquat was still happily running away.

I thought maybe Kumquat stayed chunkloaded because it was near world spawn (although not too close)? So I went and placed Cucumber on a Wheel, same setup, in the Nether. Flew around a bit, returned, Cucumber was still there. Went back to the overworld, Kumquat was still there. Cucumber was, again, still there when I returned to the Nether.

I also exited the world and loaded a different one - both Hamsters were fine. Even exited the game and returned - both were still fine.

This was in a testing instance with minimal mods and a freshly created world. So I suspect that this was either fixed in one of the subsequent updates, or it's a mod interaction of some kind.
